demoScripts/createVMAndSSH.md

```diff
@@ -0,0 +1,82 @@
+# Create an Azure Linux VM and SSH Into it
+
+## Define Environment Variables
+
+The First step in this tutorial is to define environment variables 
+
+```bash
+export MY_RESOURCE_GROUP_NAME=myResourceGroup
+export MY_LOCATION=EastUS
+export MY_VM_NAME=myVM
+export MY_USERNAME=azureuser
+export MY_VM_IMAGE=UbuntuLTS
+```
+
+# Login to Azure using the CLI
+
+In order to run commands against Azure using the CLI you need to login. This is done, very simply, though the `az login` command:
```

Comment:
No command block

Reply:
This is another one to figure out how to handle, we are not able to do a successful Az login within Innovation engine. As so, taking this out of a command block is ideal. (This was how the document was already written)